client/src/components/App/App.js

```diff
@@ -1,21 +1,74 @@
 import React, { Component } from 'react';
 import './App.css';
 import styled from 'styled-components';
-import NavBar from '../NavBar';
-
 
 class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      signUpEmail: '',
+      signUpPassword:'',
+      signUpPasswordConfirm:''
+    }
+  }
+
   render() {
     return (
       <div>     
           Sign Up <br/>
-          <input
-              type="text"
-              placeholder="First Name"/><br/> 
-          <input type="submit" value="Submit" />
+          <form>
+            <input
+              type="email"
+              name="signUpEmail"
+              placeholder="Email"
+              value={this.state.signUpEmail}
+              onChange={this.handleChange}/><br/> 
+             <input
+              type="password"
+              name="signUpPassword"
+              placeholder="Password"
+              value={this.state.signUpPassword}
+              onChange={this.handleChange}/><br/>
+            <input
+              type="password"
+              name="signUpPasswordConfirm"
+              placeholder="Confirm Password"
+              value={this.state.signUpPasswordConfirm}
+              onChange={this.handleChange}/><br/>  
+            <input 
+              type="submit" 
+              value="Submit" 
+              onClick={this.handleSubmit}/>
+          </form> 
       </div>
     );
   }
-}
 
+  handleChange = (event) => {
+    this.setState({[event.target.name]: event.target.value});
+  }
+
+  handleSubmit = (event) => {
+    console.log(this.state.signUpPassword);
+    console.log(this.state.signUpPasswordConfirm);
+    if(this.state.signUpPassword !== this.state.signUpPasswordConfirm){
```

Comment:
@amtadros If you moved  the `const { signUpEmail, signUpPassword} = this.state;` declaration to the top of this function and added `signUpPasswordConfirm` you could use `signUpPassword` and `signUpPasswordConfirm` directly instead of grabbing them off of `this.state..`.

It's also pretty handy to see every property of state that this handler uses at the top of it.